pytorch_lightning/trainer/trainer.py

```diff
@@ -129,6 +129,7 @@ def __init__(
             auto_lr_find: Union[bool, str] = False,
             replace_sampler_ddp: bool = True,
             progress_bar_callback: Optional[Union[ProgressBarBase, bool]] = True,
+            auto_scale_batch_size: Union[bool, str] = False,
```

Comment:
just thinking that it would be easier to be `Optional[str]` the making to complex with bool...